analyzers/src/SonarAnalyzer.Common/Rules/CommentKeywordBase.cs

```diff
@@ -34,25 +34,28 @@ public abstract class CommentKeywordBase : SonarDiagnosticAnalyzer
         protected const string TodoDiagnosticId = "S1135";
         protected const string TodoMessageFormat = "Complete the task associated to this '" + TodoKeyword + "' comment.";
 
-        private const string FixMeKeyword = "FIXME";
```

Comment:
Why change these? two different words, fix and me :)

Reply:
For consistency, the `Todo` was `Todo`, also two different words. I consider both as a single term for purpose of naming. It doesn't change much here and makes it a tiny bit more easier to see what is data prefix and what is meaning (DiagnosticId, MessageFormat). There were too many AbCdEffff jumps too close together.

Reply:
I think we should rename also `ToDo` to avoid having exceptions. One rule for all :)